src/client/containers/TaskTrainContainer/TaskTrainContainer.js

```diff
@@ -160,7 +160,7 @@ class TaskTrainContainer extends React.Component {
               solution={this.state.solution}
             />
             <SampleTests
-              defaultTests={task && task.test}
+              defaultTests={(task && task.testSample) || 'Sorry, tests sample are not present.'}
```

Comment:
Could you please move text to variable ?

Reply:
Yes, of course. Done.